Core/src/org/sleuthkit/autopsy/mainui/nodes/FileNode.java

```diff
@@ -18,25 +18,34 @@
  */
 package org.sleuthkit.autopsy.mainui.nodes;
 
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
Wrong logger :)

Reply:
@dsmyda DANNY!!!! :) How have you been? Well it's nice of you to pay attention! ;) And yes, @kellykelly3 we should indeed use org.sleuthkit.autopsy.coreutils.Logger instead :)

Reply:
Hey @eugene7646, I've been good! How are you doing?

And haha no problem, always happy to help! 

And I just want to say - big shoutout to the entire team, y'all are doing great stuff and handling lots of code :)